podcasts/SharingHelper+swipeButton.swift

```diff
@@ -0,0 +1,26 @@
+import UIKit
+import PocketCastsDataModel
+
+extension SharingHelper {
+    func shareLinkTo(episode: Episode, fromController: UIViewController, fromTableView tableView: UITableView, at indexPath: IndexPath) {
+        let source = tableView.swipeButton(forLabel: L10n.share, at: indexPath) ?? tableView
+        Analytics.track(.podcastShared, properties: ["type": "episode", "source": "episode_swipe_action"])
```

Comment:
I'd like any feedback about the `source` value (I haven't updated the spreadsheet yet and will do once this is accepted or we come up with a better name).

Reply:
I think this makes sense, since the other event is `episode_swipe_action_performed`.

Reply:
cc @mchowning